build/visible_to/BUILD

```diff
@@ -40,13 +40,8 @@ package_group(
         "//hack",
         "//hack/lib",
         "//hack/make-rules",
-        "//test/e2e",
-        "//test/e2e/framework",
-        "//test/e2e/kubectl",
-        "//test/e2e/workload",
-        "//test/integration/etcd",
-        "//test/integration/framework",
-        "//test/integration/kubectl",
+        "//test/e2e/...",
```

Comment:
I somewhat feel like `//test/e2e` shouldn't be allowed to depend on large swaths of the tree, but that's probably work for another day...